test/integration/emitter_factory_test.ts

```diff
@@ -37,11 +37,11 @@ export const fixture = new CloudEvent({
   [ext3Name]: ext3Value,
 });
 
-function axiosEmitter(message: Message, options?: Options): Promise<unknown> {
```

Comment:
It should be a goal to not require parameterization for this file. Can we try adding this feature without a change to this file?